schemas/logs/okta/system_log.yml

```diff
@@ -242,6 +252,9 @@ fields:
       - name: interface
         description: The third party user interface that the actor authenticates through, if any.
         type: string
+      - name: authenticatorProvider
```

Comment:
This was a mistake in the previous schema. We need to keep the field for backwards compatibility